webapp/src/containers/components/modals/actionModal.tsx

```diff
@@ -5,21 +5,28 @@ import Table from 'react-bootstrap/Table';
 import Form from 'react-bootstrap/Form';
 import './styles.css';
 
+// eslint-disable-next-line import/no-unresolved
+import {Configs} from 'types/plugin/common';
```

Comment:
there is no need to import types it should work anywhere without any explicit import/export if its defined in a file `<file-name>.d.ts`

Reply:
the types are not working in this project without importing, tried multiple fixes but it was not resolved

Reply:
Discuss with @sooraj-shukla or @muazzamqazi on this once.

Reply:
I'm approving this PR for now but make sure you discuss and fix this in a separate PR or in the same PR before merging.

Reply:
sure